mysql-test/suite/innodb/r/percona_session_temp_tablespaces_encrypt.result

```diff
@@ -93,6 +93,7 @@ SELECT * FROM INFORMATION_SCHEMA.INNODB_SESSION_TEMP_TABLESPACES WHERE ID = conn
 ID	SPACE	PATH	SIZE	STATE	PURPOSE
 ID	4294501260	./#innodb_temp/temp_4.ibt	98304	ACTIVE	INTRINSIC ENCRYTPED
 ID	4294501261	./#innodb_temp/temp_5.ibt	147456	ACTIVE	USER ENCRYPTED
+ALTER INSTANCE ROTATE INNODB MASTER KEY;
```

Comment:
This test does not test the crash situation. Some Debug sync between one connection issuing ALTER INSTANCE ROTATE and the other performing SESSION disconnection or temp table truncation would be needed.